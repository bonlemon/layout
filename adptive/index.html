<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Golden</title>
    <link rel="stylesheet" href="styles/main.css">
</head>
<body>
    <div class="wrap">
        <header class="row vertical-centerr">
            header (logo etc)
        </header>
        <div class="row middle">
            <div class="content">content</div>
            <div class="sidebar">sidebar</div>
        </div>
    </div>
    <footer class="row vertical-venter">
        footer (copyright etc)
    </footer>
</body>
</html>