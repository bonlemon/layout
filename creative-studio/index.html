<!DOCTYPE html>
<html lang="ru">
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no, maximum-scale=1">
	<title>Главная</title>
	<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i" rel="stylesheet">
	<link rel="stylesheet" type="text/css" href="css/normalize.css" />
	<link rel="stylesheet" type="text/css" href="css/style.css" />
</head>
<body>
		


<script type="text/javascript" src="js/jquery-1.11.0.min.js"></script>
<script type="text/javascript" src="js/custom.js"></script>

</body>
</html>