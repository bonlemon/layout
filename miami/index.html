<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="flex.css">
<<<<<<< HEAD
    <link rel="stylesheet" href="normalize.css">
    <link rel="stylesheet" href="mail.css">
=======
    <link rel="stylesheet" href="main.css">
>>>>>>> 07badef1f17b898af1b9d26a505dda438a5ef9d7
    <title>Document</title>
</head>
<body>
    
</body>
</html>