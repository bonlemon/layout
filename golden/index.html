<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Golden</title>
    <link rel="stylesheet" href="styles/main.css">
</head>

<body>
    <header>
        <div class="container">
            <div class="heading clearfix">
                <img src="img/logo.png" alt="Golden" class="logo">
                <nav>
                    <ul class="menu">
                        <li>
                            <a href="#">
                                Home
                            </a>
                        </li>
                        <li>
                            <a href="#">
                                Services
                            </a>
                        </li>
                        <li>
                            <a href="#">
                                Portfolio
                            </a>
                        </li>
                        <li>
                            <a href="#">
                                About
                            </a>
                        </li>
                        <li>
                            <a href="#">
                                Contact
                            </a>
                        </li>
                    </ul>
                </nav>
            </div>
            <div class="titles">
                <div class="titles__first">
                    Welcome To Our Studio!
                </div>
                <h1 class="titles__firsr">
                    It’s nice to Meet you
                </h1>
            </div>
            <a href="#" class="button">
                Tell Me More
            </a>
        </div>
    </header>

    <section id="services">
        <div class="container">
            <div class="title">
                <h2>
                    Services
                </h2>
                <p>
                    Proin iaculis purus consequat sem cure.
                </p>
            </div>
            <div class="services clearfix">
                <div class="services__item">
                    <img src="img/icon1.png" alt="E-Commerce">
                    <h3>
                        E-Commerce
                    </h3>
                    <p>
                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                    </p>
                </div>
                <div class="services__item">
                    <img src="img/icon2.png" alt="Responsive Web">
                    <h3>
                        Responsive Web
                    </h3>
                    <p>
                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt
                    </p>
                </div>
                <div class="services__item">
                    <img src="img/icon3.png" alt="Web Security">
                    <h3>
                        Web Security
                    </h3>
                    <p>
                        Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                    </p>
                </div>
            </div>

        </div>
    </section>
    <section id="portfolio">
        <div class="container">
            <div class="title">
                <h2>
                    Our portfolio
                </h2>
                <p>
                    Proin iaculis purus consequat sem cure.
                </p>
            </div>
            <div class="works clearfix">
                <img src="img/img1.jpg" alt="Работа">
                <img src="img/img2.jpg" alt="Работа">
                <img src="img/img3.jpg" alt="Работа">
            </div>
        </div>
    </section>
    <section>
        <div class="container">
            <div class="title">
                <h2>
                    About Us
                </h2>
                <p>
                    Proin iaculis purus consequat sem cure.
                </p>
            </div>

            <div class="about clearfix">
                <div class="about__item clearfix ">
                    <div class="about__text">
                        <h3>
                            July 2010
                        </h3>
                        <h3>
                            Our Humble Beginnings

                        </h3>
                        <p>
                            Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                            Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.
                        </p>
                    </div>
                    <img src="img/about1.jpg" alt="About">
                </div>
                <div class="about__item">
                    <div class="about__text clearfix">
                        <h3>
                            January 2011
                        </h3>
                        <h3>
                            Facing Startup Battles
                        </h3>
                        <p>
                            Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                            Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus
                            posuere odio in tincidunt.
                        </p>
                    </div>
                    <img src="img/about2.jpg" alt="About">
                </div>
                <div class="about__item">
                    <div class="about__text clearfix">
                        <h3>
                            December 2012
                        </h3>
                        <h3>
                            Enter The Dark Days
                        </h3>
                        <p>

                            Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                            Proin iaculis purus consequat sem cure digni.
                        </p>
                    </div>
                    <img src="img/about3.jpg" alt="About">
                </div>
                <div class="about__item">
                    <div class="about__text clearfix">
                        <h3>

                            february 2014
                        </h3>
                        <h3>
                            Our Triumph
                        </h3>
                        <p>
                            Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.
                            Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.
                        </p>
                    </div>
                    <img src="img/about4.jpg" alt="About">
                </div>
            </div>
        </div>
    </section>
    <footer>
        <div class="container">
            <div class="copyright">
                © Copyright 2014 FreebiesXpress.com
            </div>
        </div>
    </footer>
</body>

</html>